### ISSUE TITLE
How to generate his final result of 90.09 in test set？

### ISSUE THREAD

**Issue by vicdu** [anibali/eval-mpii-pose#2]
hi, anibail, how to generate his final result of 90.09 in test set with https://github.com/bearpaw/pytorch-pose 
 ?And how to generate the test set? thanks

**Comment by anibali** [comment]
In short, you can't generate test set accuracy yourself, since the ground truth annotations for the test set are not public. Test set metrics can only be obtained by submitting predictions to MPII.

What the code in this repository lets you do is easily work with predictions produced from non-Matlab models. For example, you can:

1. Evaluate validation set prediction accuracy.
2. Convert test set predictions to the format required for MPII submission.

**Comment by vicdu** [comment]
Thanks Reply! Now I have the trained model with pytorch. but how can I get the test set predictions in pytorch-pose? thanks

**Comment by anibali** [comment]
You should ask that over at pytorch-pose.

If you want to calculate validation set metrics, take the `preds_valid.mat` file produced by bearpaw's code and add a new case for it in https://github.com/anibali/eval-mpii-pose/blob/master/getExpParamsNew.m.

**Comment by vicdu** [comment]
thanks. I just wrote a code that generates mpii-test json according to test.h5 with https://github.com/umich-vl/pose-hg-train.  I plan to use your code and submit after forecast.
